am/templates/common/clusterrole.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.am.managedServiceAccount }}
+{{- if and .Values.am.managedServiceAccount .Values.am.clusterRoleRules }}
```

Comment:
Do we still need clusterRole manifest ?

Reply:
We don't need it if the user wants to put everything in the same namespace but it is better to keep the cluster role option for those users who want to:
1) Have access to different resources in other namespaces
2) Resolve property files inside gravitee.yaml using KubernetesPropertyResolver from all namespaces

Reply:
in that case ok.